recipes/chronometrist

```diff
@@ -1,6 +1,9 @@
 (chronometrist
- :repo "contrapunctus-1/chronometrist"
- :fetcher github
+ :url "https://tildegit.org/contrapunctus/chronometrist.git"
+ :fetcher git
+ :branch "production"
```

Comment:
MELPA prefers to build from your default branch (ie. development version), and version tags will be used to identify stable versions.